### PR TITLE
Update NuGet versions of all modules

### LINKED ISSUE #173: RocksDB Building error
```cs
Plugins/Storage/RocksDBStore.cs(10,30): error CS0507: 'RocksDBStore.Configure()': cannot change access modifiers when overriding 'protected' inherited member 'Plugin.Configure()' [/opt/neo-plugins/src/RocksDBStore/RocksDBStore.csproj]
```

@shargon, do you have an idea?

### PR DESCRIPTION
**Pull request by vncoelho**
closes #173

### CODE REVIEW

**Comment by vncoelho** [comment]
@shargon @erikzhang 

```cs
TransactionManager.cs(107,69): error CS0117: 'OpCode' does not contain a definition for 'PUSHBYTES64' [/opt/neo-plugins/src/RpcClient/RpcClient.csproj]
TransactionManager.cs(107,122): error CS0117: 'OpCode' does not contain a definition for 'PUSHBYTES33' [/opt/neo-plugins/src/RpcClient/RpcClient.csproj]
TransactionManager.cs(107,176): error CS0117: 'InteropService' does not contain a definition for 'Neo_Crypto_ECDsaVerify' [/opt/neo-plugins/src/RpcClient/RpcClient.csproj]
```

We still need to fix these ones.

File: src/RpcClient/TransactionManager.cs [hunk 1]
```diff
@@ -104,7 +104,7 @@ private long EstimateNetworkFee()
             foreach (var hash in hashes)
             {
                 size += 166;
-                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Neo_Crypto_ECDsaVerify, null);
+                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Crypto.ECDsaVerify, null);
             }
```
Comment (shargon) [inline src/RpcClient/TransactionManager.cs]: Also this opcodes need to be changed

Comment (shargon) [reply src/RpcClient/TransactionManager.cs]: Please check it, i am not sure if it's well estimated

Comment (vncoelho) [reply src/RpcClient/TransactionManager.cs]: I think it is good, @shargon, and well detailed!

Comment (vncoelho) [reply src/RpcClient/TransactionManager.cs]: This test is failing still: https://github.com/neo-project/neo-modules/blob/3fddad5e9b8d60211c635afe80d8c29176a582fb/tests/Neo.Network.RPC.Tests/UT_TransactionManager.cs#L124

File: src/RpcClient/TransactionManager.cs [hunk 2]
```diff
@@ -104,7 +104,14 @@ private long EstimateNetworkFee()
             foreach (var hash in hashes)
             {
                 size += 166;
-                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES64] + ApplicationEngine.OpCodePrices[OpCode.PUSHBYTES33] + InteropService.GetPrice(InteropService.Neo_Crypto_ECDsaVerify, null);
+                networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHNULL]; // message
+                using (ScriptBuilder sb = new ScriptBuilder())
+                {
+                    networkFee += ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(64).ToArray()[0]]; // signature
+                    networkFee += ApplicationEngine.OpCodePrices[(OpCode)sb.EmitPush(33).ToArray()[0]]; // pubKey
+                    size += sb.ToArray().Length;
```
Comment (vncoelho) [inline src/RpcClient/TransactionManager.cs]: Before we were not considering this in the size, right?

Comment (shargon) [reply src/RpcClient/TransactionManager.cs]: I am not sure if it's included in this 166

**Comment by cloud8little** [comment]
@vncoelho Is this ready for testing?

**Comment by vncoelho** [comment]
I think so, @cloud8little, let's update each package in order that we move forward with all of them!

**Comment by vncoelho** [comment]
It is not ready I think, even for the RPCClient, we still have some problems with the Signing UT.

File: src/RpcClient/TransactionManager.cs (hunk 1 above)
Comment (chenquanyu) [reply src/RpcClient/TransactionManager.cs]: Skip(2)

File: src/RpcClient/TransactionManager.cs (hunk 2 above)
Comment (chenquanyu) [reply src/RpcClient/TransactionManager.cs]:
```
if (witness_script.IsSignatureContract())
 {
    size += 67 + witness_script.GetVarSize();
    networkFee += ApplicationEngine.OpCodePrices[OpCode.PUSHDATA1] + ApplicationEngine.OpCodePrices[OpCode.PUSHDATA1] + ApplicationEngine.OpCodePrices[OpCode.PUSHNULL] + InteropService.GetPrice(InteropService.Crypto.ECDsaVerify, null);
 }
```
It's the varsize of invocation and verification script.

**Comment by cloud8little** [comment]
> It is not ready I think, even for the RPCClient, we still have some problems with the Signing UT.
according to @chenquanyu ; try this
Skip(2) for this 
https://github.com/neo-project/neo-modules/blob/master/tests/Neo.Network.RPC.Tests/UT_TransactionManager.cs#L122 

no change to https://github.com/neo-project/neo-modules/blob/master/tests/Neo.Network.RPC.Tests/UT_TransactionManager.cs#L124
and UT passed.

**Comment by vncoelho** [comment]
Sorry, @cloud8little, I did not understand your last comment.

File: src/RpcClient/TransactionManager.cs (hunk 2 above)
Comment (vncoelho) [reply src/RpcClient/TransactionManager.cs]: Could you proceed with the change @chenzhitong?

**Comment by chenquanyu** [comment]
Hi @vncoelho 
I have changed the fee calculation and UT in this PR #178, please help to review.

**Comment by cloud8little** [comment]
> Sorry, @cloud8little, I did not understand your last comment.

@vncoelho https://github.com/neo-project/neo-modules/pull/178/files#diff-ef6775ea0f925a895f26d4ee4f447e10L122 
Smoke Test Passed, except the UT failed, please help update. and smart contract deployment is not tested since neo-devpack-dotnet is not up-to-date and opcodes there can not be recognized by Neo "3.0.0-CI00846".  Besides, could you update a stable Neo nuget version, for now 3.0.0-CI00846 has been tested with latest neo-node a137b6f20c957706cfbbea323e2a1d18e8c79a14, in this way, we can be sure which version works well and already has been tested.

**Comment by vncoelho** [comment]
I got it, @cloud8little, check now after the great work of @chenquanyu.

File: src/RpcClient/RpcClient.csproj [hunk 3]
```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00825" />
+    <PackageReference Include="Neo" Version="3.0.0-CI00847" />
```
Comment (shargon) [inline src/RpcClient/RpcClient.csproj]: 846 in the others, we should use the last in all of them

File: src/RpcClient/TransactionManager.cs [hunk 4]
```diff
@@ -84,56 +84,49 @@ public TransactionManager MakeTransaction(byte[] script, TransactionAttribute[]
...
             foreach (UInt160 hash in hashes)
             {
-                byte[] witness_script = context.GetScript(hash);
-                if (witness_script is null || witness_script.Length == 0)
+                byte[] witness_script = null;
+                if (isEstimate)
                 {
-                    try
+                    // assuming the witnesses are basic Signature Contract
+                    KeyPair one = new KeyPair(new byte[31].Concat(new byte[] { 1 }).ToArray());
```
Comment (shargon) [inline src/RpcClient/TransactionManager.cs]:
```suggestion
                    var dummyKey = new byte[32];
                    dummyKey[31] = 0x01;
                    KeyPair one = new KeyPair(dummyKey);
```

Comment (chenquanyu) [reply src/RpcClient/TransactionManager.cs]: 👍This will be faster.

**Comment by vncoelho** [comment]
@shargon,

all `.dlls` up-to-date with this patch now:

image

**Comment by vncoelho** [comment]
@shargon, we plan to setup some nodes of the privnet with RocksDb and other with LevelDb while other ones with Local Memory.
That will be good some tests.

**Pull request merged**